### ISSUE TITLE
Analytics Opt-in: "You can read all our terms here" link is not clickable

### ISSUE THREAD

**Issue by anoadragon453** [vector-im/element-x-android#1174]
### Steps to reproduce

1. Go to User Settings -> Analytics
2. See the text "You can read all our terms here."
3. Attempt to click the text and instead Analytics is turned on.

I could see lots of people complaining about this :)

#### What did you expect?

The analytics terms and conditions to appear on the screen.

#### What happened instead?

I accidentally opted into sending analytics data :boom: 

### Your phone model

Google Pixel 4a 5G

### Operating system version

Android 13.0

### Application version and app store

Compiled from source; commit 5a85459ec2e8d1d3ad2d6d68ffbb7eb6b250ec1e

### Homeserver

element.io

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by bmarty** [comment]
I remember having reported that this was missing, but I cannot find out where.

I think we were waiting for the URL, can you help on this @giomfo ?

**Comment by jmartinesp** [comment]
> I think we were waiting for the URL

There is already an URL in the `AnalyticsOptInNode` if I'm not mistaken. It just wasn't added here.

**Comment by bmarty** [comment]
Yes, thanks Jorge. I have talked to Giomfo, but forgot to report the status here.
